app/models/audit.rb

```diff
@@ -0,0 +1,31 @@
+class Audit < ApplicationRecord
+  belongs_to :user
+  belongs_to :organization
+  belongs_to :storage_location
+  belongs_to :adjustment, optional: true
+
+  include Itemizable
+  include Filterable
+  scope :at_location, ->(location_id) { where(storage_location_id: location_id) }
+
+  accepts_nested_attributes_for :adjustment
+
+  enum status: { in_progress: 0, confirmed: 1, finalized: 2 }
+
+  validates :storage_location, :organization, presence: true
+  validate :line_item_items_exist_in_inventory
+  validate :line_item_items_quantity_is_positive
+  validate :storage_locations_belong_to_organization
```

Comment:
I'm not sure this is necessary -- were you able to adjust another organization's storage locations by manipulating the params (Burpsuite or other means)? If so, that's a security bug and if you could create an issue with the steps to replicate that, that would be really helpful. :bowing_man: 

I think this should probably be checked in the Adjustment itself, if it's necessary, since it's the `adjust!` method that actually changes the inventory counts. Was this validation created because you noticed an issue while testing it?

Reply:
I got an issue once. I was not able to replicate that.
I got some invalid `storage_location` to the controller which is added as it is as there was no check before. When I tried to edit that, the edit page did not show any storage location as that id is invalid.

Reply:
Also, yeah like you said, we are not adjusting anything while audit creation workflow. So this check would be unnecessary. I added this keeping in mind that if an audit is created with an invalid storage location and that audit is confirmed, and then while adjusting if we find out that the storage location is invalid, there is no way to edit audit, as the audit would be in `confirmed` state. So it means, redoing of audit again

Reply:
I don't think the model for `audit` should know anything about/validate the relationship between storage location and organization